autogpt_platform/backend/backend/server/integrations/router.py

```diff
@@ -29,7 +29,7 @@ class LoginResponse(BaseModel):
 
 
 @router.get("/{provider}/login")
-async def login(
```

Comment:
All these routes use db_manager inside and doesn't use prisma, so replace all of them to sync def